public/controllers/agent/components/register-agent.js

```diff
@@ -473,7 +485,14 @@ export const RegisterAgent = withErrorBoundary(
 
       const guide = (
         <div>
-          {this.state.selectedOS && (
+          {(!this.state.needsPassword && this.state.wazuhPassword=== "") ? (
```

Comment:
**thought**: I am not sure about this check to display the callout. Maybe we should have a state when the request fails.

Reply:
hmmm... I'm going to check this

Reply:
**review**:

I tested the PR and when there isn't a configurated password, the callout is displying:
![image](https://user-images.githubusercontent.com/34042064/124456603-2f60d580-dd8b-11eb-8ab7-3eb856f9e2f0.png)